src/main/scala/nl.knaw.dans.easy.multideposit/parser/Headers.scala

```diff
@@ -34,6 +37,8 @@ object Headers {
       audioVideoHeaders
   }
 
+  logger.info(s"VALID HEADERS $validHeaders")
```

Comment:
👍 I like this addition.
However, make it `debug` rather than `info`. With `info`, the datamanager will see this log everytime the tool is used. That's not needed, I think.